charts/firefly-evmconnect/Chart.yaml

```diff
@@ -6,7 +6,7 @@ description: |
   transaction state.
 type: application
 version: 0.7.0
-appVersion: "0.1.3"
+appVersion: "1.4.4"
 
 maintainers:
   - name: hfuss
```

Comment:
update to @onelapahead please

Reply:
https://github.com/hyperledger/firefly-helm-charts/pull/76/commits/205f20754d1f00a68130ca2234e50bf3db86aca8